config/environments/development.rb

```diff
@@ -67,6 +67,9 @@
   # Raise error when a before_action's only/except options reference missing actions.
   config.action_controller.raise_on_missing_callback_actions = true
 
+  # 
```

Comment:
Forgot to write comment
```suggestion
  # Default url required by devise's installation.
  # In production, :host should be set to actual application's host.
```